### PR TITLE
GSLUX-749: Use new v4 auth form component

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-749

### Description

This PR needs https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/pull/165 from v4 to work.

### CODE REVIEW

File: geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js [hunk 1]
```diff
@@ -284,50 +281,71 @@ exports.prototype.setUserInfo = function(
  * @return {string} The username.
  */
 exports.prototype.getUsername = function() {
-  return this.username;
```
Comment (tkohr) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js:287]: Can we get rid of the old properties?

File: geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js [hunk 2]
```diff
@@ -284,50 +281,71 @@ exports.prototype.setUserInfo = function(
...
+  const userManagerStore = useUserManagerStore()
+  const { currentUser } = storeToRefs(userManagerStore)
```
Comment (tkohr) [inline geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js:285]: Could it make sense to instantiate these once in the constructor?

**Review by tkohr — COMMENTED** [review]
Thanks @AlitaBernachot, nice integrated feature :smiley:. Authentication works fine here, I'm just getting some CORS errors when trying to connect with v4 to the v3 backend.

File: geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js (hunk 2 above)
Comment (AlitaBernachot) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js]: done

File: geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js (hunk 1 above)
Comment (tkohr) [reply geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js:287]: right, I guess it's not worth it

**Pull request merged**